### PR TITLE
Overflow a11y pass

### PR DESCRIPTION
**Pull request by micahgodbolt**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes 1, 2, 3, 6 of  #7761
Bug1: [Accessibility][Programmatic access]: Buttons must have discernible text."
Bug2: [Accessibility][Programmatic access]: Incorrect Aria-label is defined for 'Add' button. "
Bug3: [Accessibility][Programmatic access]: Narrator does not read for tooltip as it is not properly defined in code.
Bug6: [Accessibility][Programmatic access]: User does not get the information related to orientation of the menus.
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/10499)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/OverflowSet/examples/OverflowSet.Custom.Example.tsx [hunk 1]
```diff
@@ -51,27 +52,6 @@ export class OverflowSetCustomExample extends React.PureComponent {
           }
         ]}
         overflowItems={[
-          {
```
Comment (natalieethell) [inline packages/office-ui-fabric-react/src/components/OverflowSet/examples/OverflowSet.Custom.Example.tsx]: Is this removal intentional?

Comment (micahgodbolt) [reply packages/office-ui-fabric-react/src/components/OverflowSet/examples/OverflowSet.Custom.Example.tsx]: redundant commands in our example. from an A11y standpoint the menu was confusing. better to just not have that in the example.

**Comment by natalieethell** [comment]
Going to close and re-open to kick off another build - looks like the operation was cancelled

**Comment by micahgodbolt** [comment]
yeah, not 100% sure why this isn't passing.

**Comment by micahgodbolt** [comment]
Ok, this is finally passing! Snaps in experimental were the issue. @joschect can you give me a sign off here?
